content/user/first_steps/secure_access.md

```diff
@@ -24,13 +30,17 @@ of them should be available.
 
 If your hosting company does not offer some mechanism for secure file transfer, then they are most likely not PCI Compliant either, and you should be choosing a different hosting company who takes security seriously.
 
+## Secure access to your Admin
+
+Be sure your `admin/includes/configure.php` file has all URL settings using `https`.  This includes `HTTP_SERVER`, `HTTP_CATALOG_SERVER` and `HTTPS_CATALOG_SERVER`, and if it exists, `HTTPS_SERVER`. 
 
-## Secure cPanel Access 
+Use an admin username other than `admin` (or `nimda`).  Make it hard to guess.
 
-## Secure access to your Admin
+## Secure cPanel Access 
 
-## Secure access to your database 
+Just because you run an SSL on your site doesn't mean your cPanel access is secure.  Look for the padlock in your browser's address bar, and tell your hoster to fix it if it's not there! 
```

Comment:
Should we drop the term `cPanel` to avoid confusion with the brand-named web-mgmt platform by that name, vs the store's admin portal?

Reply:
No this is specific to cPanel.  This is the cPanel section, there is a separate "secure access to your admin" section above it.

Reply:
Okay. I hadn't looked at the image.

Still, I'm inclined to remove this cPanel section, unless we're going to document the same about every other hosting control panel platform out there too. Is it still that prolific a problem that hosts aren't securing their cPanel access by default that "we" need to take responsibility to tell storeowners their hosts are incompetent?

Reply:
This is surprisingly common in my experience.  And cPanel is really dominant, there are not a ton of other control panels out there.